Subdivision_method_3/doc/Subdivision_method_3/Subdivision_method_3.txt

```diff
@@ -127,12 +127,12 @@ There is only one line deserving a detailed explanation:
 
 \code{.cpp}
 
-Subdivision_method_3::CatmullClark_subdivision(pmesh, d);
+Subdivision_method_3::CatmullClark_subdivision(pmesh, params::number_of_iterations(d));
```

Comment:
`CGAL::parameters` (same for others)

Reply:
We are in 4.11, it should be `CGAL::Polygon_mesh_processing::parameters` but I'm using the alias from the example.